vent/core/rq_worker/watch.py

```diff
@@ -186,6 +189,26 @@ def file_queue(path, template_path="/vent/"):
             link_name = config.get(section, 'link_name')
             name_maps[link_name] = image_name.replace(':', '-').replace('/', '-')
             # doesn't matter if it's a repository or registry because both in manifest
+            if 'dump' in path:
```

Comment:
this should check for specific path types, the `vent.template` of each plugin should be handling that.